optional_components/speech_api/Dockerfile

```diff
@@ -7,9 +7,13 @@ ARG HOME_DIR=/usr/src/${NAME}
 ARG PORT=8001
 ARG WHISPER_MODEL_DIR=/whisper_models
 ARG PREFERRED_MODEL=small
+ARG PIPER_MODELS_DIR=/models/piper
```

Comment:
We should think about multistage build in the future to reduce the size of the instance